pkg/minishift/registration/utils.go

```diff
@@ -60,6 +61,10 @@ func RegisterHostVM(host *host.Host, param *RegistrationParameters) error {
 		if err := registrator.Register(param); err != nil {
 			return err
 		}
+		minishiftConfig.InstanceConfig.IsRegister = true
+		if err := minishiftConfig.InstanceConfig.Write(); err != nil {
```

Comment:
The method could return the bool which then gets written into persistent config.